experimental/sdk/webpubsub-socketio-extension/src/SIO/components/web-pubsub-adapter.ts

```diff
@@ -384,6 +393,30 @@ opts = ${toOptionsString(opts)}, error = ${e}`);
     return result;
   }
 
+  /**
+   * Generates OData filter string for Web PubSub service from a set of rooms and a set of exceptions
```

Comment:
`rooms` and `OData` don't exist here. And L399 should be `expcets` rather than `except`